src/views/main.js

```diff
@@ -47,7 +48,11 @@ const mainLayout = data => `
     <!-- End Google Tag Manager -->
     </head>
   <body>
-    ${['landing', 'notFound'].includes(data.partial.name) ? landingHeader(data) : mainHeader(data)}
+    ${
+      showLandingHeaderFor.includes(data.partial.name)
+        ? landingHeader(data)
+        : mainHeader(data)
+    }
```

Comment:
Nice!  I mentioned in one of @Vinnl 's PRs for error page that we should eventually create a new layout specific to non-auth pages (see MNTOR-1254), but this is a good improvement for the meantime, thanks!